scipy-1.0/paper.tex

```diff
@@ -570,7 +570,9 @@ \subsection*{Project scope}
 In terms of software systems and architecture, SciPy's scope matches NumPy's:
 algorithms for in-memory computing on single machines, with support for a wide
 range of data types and process architectures. Distributed computing and support
-for graphics processing units (GPUs) are explicitly out of scope.
+for graphics processing units (GPUs) were explicitly out of scope at the 1.0
+release point, but this has been revised in our roadmap at the time of writing
+(see below).
```

Comment:
Since it's far below, perhaps reference the discussion section.

Reply:
changed to "see Discussion," thanks